user/NickDraper/test script 2 copy.py

```diff
@@ -1 +1,2 @@
-print "hi there"
\ No newline at end of file
+from __future__ import print_function)

```

Comment:
I don't think the bracket at the end is meant to be there? Similarly for the files below?